layers/cmd/ingest_csv.go

```diff
@@ -30,6 +31,97 @@ import (
 	"github.com/cloudprivacylabs/opencypher/graph"
 )
 
+type CSVIngester struct {
+	Step
+	BaseIngestParams
+	StartRow     int
+	EndRow       int
+	HeaderRow    int
+	ID           string
+	IngestByRows bool
+}
+
+func (ci CSVIngester) Run(pipeline *PipelineContext) error {
+	layer, err := LoadSchemaFromFileOrRepo(pipeline.Context, ci.CompiledSchema, ci.Repo, ci.Schema, ci.Type, ci.Bundle)
```

Comment:
Do the same init logic as CSVExporter. Intialize only once. Store the layer in ci